discovery-provider/Caddyfile

```diff
@@ -15,6 +15,23 @@
     reverse_proxy {$ROOT_HOST:"openresty:5000"}
 }
 
+# Add a second HTTPS domain for unproxied requests
+{$audius_discprov_url_unproxied} {
+    {$CADDY_TLS}
```

Comment:
we want to remove this line so it does (default) letsencrypt

Reply:
ah yes